subsys/bluetooth/services/ots/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_sources_ifdef(
+  CONFIG_BT_GATT_OTS_SRV
+  ots_srv.c
+  ots_srv_l2cap.c
+  ots_srv_obj_manager.c
+  ots_srv_oacp.c
+  ots_srv_olcp.c)
```

Comment:
The directory is already ots, not sure why we need it prefixed to all files, though this can be fixed later.

Reply:
I'm planning to rename in the following way:

ots_srv.c -> server.c
ots_srv_l2cap.c -> l2cap.c
ots_srv_obj_manager.c -> obj_manager.c
ots_srv_oacp.c -> oacp_server.c
ots_srv_olcp.c  -> olcp_server.c

Plus headers:
ots_internal.h -> server_internal.h
ots_srv_l2cap_internal.h -> l2cap_internal.h
ots_srv_oacp_internal.h -> oacp_internal.h
ots_srv_olcp_internal.h -> olcp_internal.h
ots_srv_obj_manager_internal.h -> obj_manager_internal.h

Reply:
Well, I see some objections to my proposal so I am leaving it out from the scope of this PR as you suggested. This will be addressed in future PRs.